lib/structures/Message.js

```diff
@@ -64,41 +64,14 @@ class Message extends Base {
         } else {
             this.member = null;
         }
+        // Types 1 to 5 are never constructed (only exist in group channels)
         if(this.type === 0 || this.type === undefined);
-        else if(this.type === 1) {
-            data.content = `${this.author.mention} added <@${data.mentions[0].id}>.`;
-        } else if(this.type === 2) {
-            if(this.author.id === data.mentions[0].id) {
-                data.content = `@${this.author.username} left the group.`;
-            } else {
-                data.content = `${this.author.mention} removed @${data.mentions[0].username}.`;
-            }
-        } else if(this.type === 3) { // (╯°□°）╯︵ ┻━┻
-            if(data.call.ended_timestamp) {
-                if((!this.channel.lastCall || this.channel.lastCall.endedTimestamp < Date.parse(data.call.ended_timestamp))) {
-                    data.call.id = this.id;
-                    this.channel.lastCall = new Call(data.call, this.channel);
-                }
-                if(data.call.participants.includes(client.user.id)) {
-                    data.content = `You missed a call from ${this.author.mention}.`;
-                } else {
-                    data.content = `${this.author.mention} started a call.`;
-                }
-            } else {
-                if(!this.channel.call) {
-                    data.call.id = this.id;
-                    this.channel.call = new Call(data.call, this.channel);
-                }
-                data.content = `${this.author.mention} started a call. — Join the call.`;
-            }
-        } else if(this.type === 4) {
-            data.content = `${this.author.mention} changed the channel name: ${data.content}`;
-        } else if(this.type === 5) {
-            data.content = `${this.author.mention} changed the channel icon.`;
-        } else if(this.type === 6) {
+        else if(this.type === 6) {
             data.content = `${this.author.mention} pinned a message to this channel. See all the pins.`;
         } else if(this.type === 7) {
             data.content = SystemJoinMessages[~~(this.createdAt % SystemJoinMessages.length)].replace(/%user%/g, this.author.mention);
+        } else if(this.type === 8 || this.type === 9 || this.type === 10 || this.type === 11) {
```

Comment:
Should not be included in the PR. Should be in a separate PR (#491 is one of them).